src/server/infrastructure/__tests__/stream.provider.spec.ts

```
@@ -7,7 +7,7 @@ import { JetStreamApiError, RetentionPolicy, StorageType } from '@nats-io/jetstr
 import { ConnectionProvider } from '../../../connection';
 import { StreamKind } from '../../../interfaces';
 import type { JetstreamModuleOptions, StreamConfigOverrides } from '../../../interfaces';
-import { DEFAULT_EVENT_STREAM_CONFIG, internalName } from '../../../jetstream.constants';
+import { DEFAULT_DLQ_STREAM_CONFIG, DEFAULT_EVENT_STREAM_CONFIG, internalName } from '../../../jetstream.constants';
```

ESLint

[error] Replace `·DEFAULT_DLQ_STREAM_CONFIG,·DEFAULT_EVENT_STREAM_CONFIG,·internalName·` with `⏎··DEFAULT_DLQ_STREAM_CONFIG,⏎··DEFAULT_EVENT_STREAM_CONFIG,⏎··internalName,⏎`

(prettier/prettier)